portfolio/src/main/webapp/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
You shouldn't check this into git

Reply:
I'm not sure what change to make here I can't locate this file in the webapp folder

Reply:
Files starting with "." are hidden.  You should still be able to remove it with "git rm src/main/webapp/.gitignore"

Reply:
Resolving because I removed this file from git (will reflect after I push)